### ISSUE TITLE
Identify all Schedule D's

### ISSUE THREAD

**Issue by ajhoekst** [Dance-Data-Project/smith-capstone-23#6]
On Form 990 (and possibly Form 990EZ), there is a checkbox as to whether a Schedule D is required. Does this checkbox always match the presence of a Schedule D? We don't know.

Within the XML file, all Schedule D should be enclosed in a <IRS990ScheduleD> (or possibly a <IRS990EZScheduleD> or similar) block.

**Comment by ajhoekst** [comment]
@raevard raised an important point on this in our call today (Feb. 14)

Schedule D contains more than just endowment information. Presence of Schedule D does not indicate presence of an endowment. Companies *should* check the endowment checkbox on their 990, but it's possible they might not.

By the end of this task, we should know the following about the filings in our archive:

1. Which companies and years include endowment data in Schedule D
2. Did any company include endowment data but miss the checkbox on the 990?
3. Did any company check the box on the 990 but not include endowment data in Schedule D?

**Comment by raevard** [comment]
As a note, I have already worked preliminarily on questions 2 and 3; the tables in the knitted HTML that'll be included in my next push will illustrate that.  I did not explicitly extend to counting which companies and years included schedule D, but it will be an easy extension.  However, keep in mind if you look currently, the counts will be different than later, as I noticed at one point while working on this that when amended 990s were included the amount of boxes checked on the 990 increased.
